ncclient/operations/edit.py

```diff
@@ -152,6 +154,8 @@ def request(self, confirmed=False, timeout=None, persist=None):
                 sub_ele(node, "confirm-timeout").text = timeout
             if persist is not None:
                 sub_ele(node, "persist").text = persist
+        if not (confirmed and persist) and persist_id:
+            sub_ele(node, "persist-id").text = persist_id
```

Comment:
@muffizone, unless I'm mistaken the code as it is will ignore `persist-id`, but not return an error if an incorrect set of values is provided. For example, it may be desirable to generate an error/exception if each of `confirmed`, `persist` and `persist-id` are specified.

Thoughts?

Reply:
Yes makes sense to me.

I have raised the OperationError for now if the same commit session is passing confirmed, persist and persist-id.

Reply:
Hey @einarnn ,

Anything else needed for this to close ?

Reply:
This is ok, thanks! Sorry for delay.